build.sbt

```diff
@@ -20,19 +18,49 @@ version := "1.1-SNAPSHOT"
 
 scalaVersion := "2.11.11"
 
-javacOptions ++= Seq("-source", "1.7", "-target", "1.7")
+crossScalaVersions := Seq("2.11.11", "2.12.3")
+
+def scalacOptionsVersion(scalaVersion: String): Seq[String] = {
+  Seq() ++ {
+    // If we're building with Scala > 2.11, enable the compile option
+    //  switch to support our anonymous Bundle definitions:
+    //  https://github.com/scala/bug/issues/10047
+    CrossVersion.partialVersion(scalaVersion) match {
+      case Some((2, scalaMajor: Int)) if scalaMajor < 12 => Seq()
+      case _ => Seq("-Xsource:2.11")
```

Comment:
Is this necessary in Firrtl? It's necessary in chisel code using anonymous Bundles, but do the options have to be the same?

I ask because I actually want to add the "-Xsource:2.12" option when we compile with 2.11.11